src/main.cpp

```diff
@@ -291,10 +291,6 @@ static std::optional<struct timespec> get_boottime()
   if (!get_uint64_env_var("BPFTRACE_PERF_RB_PAGES", bpftrace.perf_rb_pages_))
     return false;
 
-  if (!get_uint64_env_var("BPFTRACE_MAX_TYPE_RES_ITERATIONS",
-                          bpftrace.max_type_res_iterations))
-    return 1;
```

Comment:
This should just change the return to `return false`, not remove the entire block, shouldn't it?

Reply:
Ah, didn't see that, right. Thanks for noticing and fixing this!